src/components/NoResults.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
Looks like you did it again.  I believe this file was added by accident.  Please start from a clean checkout.